### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by darlopvil** [canton7/SyncTrayzor#703]
Just want to know if this will have updates or not. Program suddenly crushed and don't respond anymore. Had to uninstall it.

**Comment by laurinkeithdavis** [comment]
I think it's a fair question and one that I've wondered about myself as a new user to Syncthing / SyncTrayzor, as the latter has not had a release in going on a year.

**Comment by tomasz1986** [comment]
SyncTrayzor is a wrapper for the official Syncthing binary. It also allows the official binary to autoupdate itself, so it's not that it needs to be updated with it. This is different e.g. from the Android releases, which tie the wrapper to a specific version of the Syncthing binary, and both need to be updated at the same time.

@[darlopvil](https://github.com/darlopvil) You may want to open the Event Viewer and check whether there's any specific information there about the crash. If you find any, please post it here.

**Comment by laurinkeithdavis** [comment]
I'm not sure about @darlopvil, but that was not my point. 

I understand it does not **have** to be updated, but any application that is not updated periodically is almost inherently not "active" (for many reasons, including security, stability, features, etc.) I'm placing a high stake in this product, as being a Windows user, Syncthing was not near as attractive a solution to me without SyncTrayzor. I just abandoned Resilio Sync after over 5 years of heavy use because of a lack of development (it is starting to have serious performance degradations), although I follow this repo to watch for activity, that activity has not been high (which could be good or bad) and the fact that there have been no commits in 9 months is not likely caused by no issues or no feature needs in my experience.

**Comment by tomasz1986** [comment]
Yeah, but even if SyncTrayzor does stop working at some point (which isn't necessary that likely, considering that Windows itself is the king when it comes to backwards compatibility, with known cases of people/companies running 20+ year old software on it perfectly fine), you can always just uninstall it and run the official Syncthing binary re-using the exact same config out of the box. If you need specific features, such as tray icons and such, there's also https://github.com/Martchus/syncthingtray, which does receive frequent updates.

**Comment by laurinkeithdavis** [comment]
Yes, I know that I can run Syncthing binary without SyncTrayzor, but I don't want to (for various reasons that I don't have time to document right now). I did not know about https://github.com/Martchus/syncthingtray, I'll keep an eye on it.

**Comment by laurinkeithdavis** [comment]
I know that I can run Syncthing without SyncTrayzor, but the point is, that I don't want to (for reasons I don't have time to document right now) - SyncTrayzor was what got me over the hump in my decision to move from Resilio. I did not know about https://github.com/Martchus/syncthingtray, I will keep an eye on that one as well.

**Comment by ryanwwest** [comment]
@laurinkeithdavis If you've got time now, I'd love to hear why you don't want to use Syncthing without SyncTrayzor and how you currently use Syncthing on Windows. I'm concerned about the same thing (lack of development coming on 2 years now) and am looking for alternatives (syncthingtray looks neat and was last updated 2 weeks ago, so I'll look into it).

**Comment by laurinkeithdavis** [comment]
> 

Honestly, I can't remember my exact reasons now. I just remember finding the Syncthing config and management process cumbersome previously, but with SyncTrayzor, not so much. I've been using them together now for over a year and the process is working great.
